mandatory/2-function-creation.js

```diff
@@ -6,7 +6,13 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+  if (num % 2 == 0 && num <= 100 && typeof(num) === "number"){
+    return true;
+  } else {
+    return false;
+  }
+}
```

Comment:
This works fine. You could also do it without using if/else. 
`return(some expression)`
will return true or false depending on whether the expression is true or false

Reply:
Thanks Karen I've tried the method and its work! cool!